assets/model_monitoring/components/src/data_quality_statistics/compute_data_quality_statistics.py

```diff
@@ -6,16 +6,16 @@
 from pyspark.context import SparkContext
 from pyspark.sql.functions import col
 from pyspark.sql.session import SparkSession
-from pyspark.sql.types import StructType, StructField, StringType
+from pyspark.sql.types import DoubleType, LongType, StructType, StructField, StringType
 import pyspark
-import pyspark.pandas as ps
 
 
 # Init spark session
 sc = SparkContext.getOrCreate()
 spark = SparkSession(sc)
 
-supported_datatype_for_max_min_value = ["IntegerType()", "DoubleType()", "ByteType()", "LongType()", "FloatType()"]
+supported_datatype_for_max_min_value = ["IntegerType()", "DoubleType()", "ByteType()",
```

Comment:
I think our PR has some dependencies on each other: 

For example, this list and the function  get_features_for_max_min_calculation can be replaced with the numerical_columns from my PR